client/src/App.tsx

```diff
@@ -1,35 +1,13 @@
-import { useState } from "react";
-import reactLogo from "./assets/react.svg";
-import viteLogo from "/vite.svg";
-import "./App.css";
+import FlavorThumbnail from "./components/FlavorThumbnail";
 
-function App() {
-  const [count, setCount] = useState(0);
+const flavor = {
+  name: "Vanilla",
+  price: 0,
+  image: "/strawberry-icecream.jpg",
+};
 
-  return (
-    <>
-      <div>
-        <a href="https://vitejs.dev" target="_blank">
-          <img src={viteLogo} className="logo" alt="Vite logo" />
-        </a>
-        <a href="https://react.dev" target="_blank">
-          <img src={reactLogo} className="logo react" alt="React logo" />
-        </a>
-      </div>
-      <h1>Vite + React</h1>
-      <div className="card">
-        <button onClick={() => setCount((count) => count + 1)}>
-          count is {count}
-        </button>
-        <p>
-          Edit <code>src/App.tsx</code> and save to test HMR
-        </p>
-      </div>
-      <p className="read-the-docs">
-        Click on the Vite and React logos to learn more
-      </p>
-    </>
-  );
+function App() {
+  return <FlavorThumbnail flavor={flavor} />;
```

Comment:
we should probably remove it from App component, likely to conflict w/ some other PRs otherwise

Reply:
For sure, good call

Reply:
We also need to make sure that someone who wasn't on our team reviews - that's the convention that we're working with!